### PR TITLE
CF_Slicing.py

### PR DESCRIPTION
**Pull request by Misseswaddles**
Completed W3 slicing lab excercise.

### CODE REVIEW

**Comment by Misseswaddles** [comment]
Adding list_lab.py from Week3. All feedback is greatly appreciated!!

File: students/CarolF/module03/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    print("Begin tasks for Series2_List_Activity//////////////////")
+    displayList(fruityList)
+    print("Begin tasks for Series3_List_Activity//////////////////")
+    userDelPreferences(fruityList)			
+
+
+def displayList(alist): #series 2 list tasks
+    """"Uses list created from makeList method, removes last value, prints result, asks user to provide a fruit to delete and removes it."""	
+    listToModify = alist
```
Comment (summerela) [inline students/CarolF/module03/list_lab.py]: why are you giving alist a new name here?

File: students/CarolF/module03/list_lab.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    
+    print("initial list with disliked fruits removed: ", listToEdit)  
+    reverseAndDelete(listToEdit)
+
+
+
+if __name__ == "__main__" :
+    series1_list = ["Apples", "Pears","Oranges","Peaches"]
+    makeList(series1_list)
+
```
Comment (summerela) [inline students/CarolF/module03/list_lab.py]: Your main function doesn't call any of the functions that ask the user for input. So, this basically just prints out the list..

File: students/CarolF/module03/slicing.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    assert everyOtherItemRemoved(a_tuple)==(2,13,5)
+    assert firstAndLast4Removed(veryLongString)==" savr ogSr"
+    assert firstAndLast4Removed(new_tuple)==(5,7,9) 
+    assert reverseOrder(new_tuple)==(14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1)
+    assert reverseOrder(veryLongString)=="!gnirtS gnol yrev a si sihT"
+    assert reverseOrder(a_list)==[32, 5, 12, 13, 54, 2]
+    assert thirdScramble(a_list)== [13, 12, 5, 32, 2, 54]
+    assert thirdScramble(a_string)=='is a sstringthis '
+    assert thirdScramble(new_tuple)==(6, 7, 8, 9, 10, 11, 12, 13, 14, 1, 2, 3, 4, 5)
+
```
Comment (summerela) [inline students/CarolF/module03/slicing.py]: Nice job on this!

**Comment by Misseswaddles** [comment]
Hi Summer,

Thanks so much for the feedback. I want to make sure I am aware of the things I ought to fix in my list_lab.py submission. Mainly, I should have “main” call the functions below, I also see I ought to return the values correctly and watch for unnecessarily changing variable names.

I am working to catch up on Mail room so once I complete that I will try to go back and edit my work.

Thanks for all the feedback, it is incredibly helpful!

Best,
Carol

**Pull request merged**